integration-test/features/services.feature

```diff
@@ -2,18 +2,54 @@ Feature: All about Services
 
   Background:
     Given GPS running
-
+    
+  Scenario: create a new service
+    Given the body of the request for a service with id "service-2"
+    When the client requests the creation of a service
+    Then the client receives status code of 201
+    
+  Scenario: create a new service with a bad body request
+    Given the bad body of the request for a service with id "service-3"
+    When the client requests the creation of a service
+    Then the client receives status code of 400
+    
+  Scenario: update a service
+    Given the updated body of the request for a service with id "service-2"
+    When the client requests a service update
```

Comment:
Create a common step for create, update e delete 
ex. `When the client requests a <creation|update|....> service`